### ISSUE TITLE
Decide on predictors

### ISSUE THREAD

**Issue by troettge** [troettge/Transparency-Ling#4]
They did not preregister any predictors. 

Possible predictors we are interested in:
- time (before after replication crisis)
- subfield (Q: how would we do that?)

Melanie found this: metadata framework on linguistic subfields: http://www.language-archives.org/REC/field.html

Possibly useful for assessing subfields!

Here is the list of journals and how they are coded in Scopus: https://www.internauka.org/en/scopus-list

Bottom line: the two codes (1203 and 3310) mostly overlap. So they are useless for classifying subfields. BUT those codes that are not Linguistic might be useful, because many journals have additional codes for e.g. Computer Science, Neuroscience, Psychology
